redesign/test.py

```diff
@@ -0,0 +1,6 @@
+import dataprofiler as dp
```

Comment:
If we are going to do tests, let's do actual unittests. and rename the file to match that which the tests are reflecting.